src/pokete/classes/difficulty.py

```
@@ -59,14 +69,28 @@ def get_style_synergy(self):
         relief = avg_res * 0.1
         return 1.0 - relief
 
-    def report_battle_detailed(self, rating, _turns, items_used, player_start_hp, player_end_hp, unique_moves, *, trainer_name=None, won=True):
-        """Processes high-fidelity battle telemetry for Style and Nemesis logic."""
+    def report_battle_detailed(self, rating, turns, items_used, player_start_hp, player_end_hp, unique_moves, *,
```

Ruff (0.14.14)

[warning] Unused method argument: `turns`

(ARG002)